setup.py

```diff
@@ -28,6 +24,7 @@
     author_email='russell@keith-magee.com',
     url='http://pybee.org/voc',
     packages=find_packages(exclude=['docs', 'tests']),
+    python_requires='==3.4',
```

Comment:
I think this should probably be `>=3.4`, because this is about the Python version you're running VOC, right?